pipeline/azure-pipelines.yml

```diff
@@ -293,8 +295,17 @@ jobs:
     displayName: 'Publish Artifact: vsix'
     inputs:
       ArtifactName: vsix
-    condition: succeededOrFailed()  # always publish artefacts - helps with diagnosing failures
-  
+    condition: succeededOrFailed()  # always publish artifacts - helps with diagnosing failures
+    
+  - task: JfrogCliV2@1
+    displayName: 'Publish Artifact vsix on repox'
```

Comment:
Test the full build with sbom

Reply:
Pipeline:
https://dev.azure.com/sonarsource/DotNetTeam%20Project/_build/results?buildId=97560&view=results

Repox artifacts:
https://repox.jfrog.io/ui/repos/tree/General/sonarsource-public-builds/org/sonarsource/sonarlint/visualstudio/sonarlint-visualstudio-2022/8.3.0.97560